### DISCUSSION TITLE
No highlights for jsx files

### DISCUSSION THREAD

**Discussion by aminehmida** [helix-editor/helix#1899] [Q&A]
I am using the latest version: helix 22.03 (d4e45fd4)

There is no highlighting if I open a jsx file. I have installed all the grammers as instructed in the readme.
Also, if I rename the file to .tsx all works perfectely.

Please let me know if you want me to open an issue.

Thank you for releasing this awosome tool 😊

**Comment by archseer** [comment]
Looks like there's a definition for `tsx` but none for `jsx`. Can you open a PR? https://github.com/helix-editor/helix/blob/84e799f0e4fdbae1dd574439f5b9776ce8ea3f8e/languages.toml#L308-L316=

**Reply by aminehmida** [reply]
PR opened: https://github.com/helix-editor/helix/pull/1906

**Reply by TornaxO7** [reply]
@aminehmida is your question here still unanswered? Maybe you can mark your question as "answered" if it's fixed :)
